src/ORM/ResultSet.php

```diff
@@ -380,12 +380,14 @@ protected function _calculateColumnMap($query)
         $map = [];
         foreach ($query->clause('select') as $key => $field) {
             $key = trim($key, '"`[]');
-            if (strpos($key, '__') > 0) {
-                $parts = explode('__', $key, 2);
-                $map[$parts[0]][$key] = $parts[1];
-            } else {
+
+            if (strpos($key, '__') <= 0) {
```

Comment:
I just refactored the `if..else` to have no `else`, no changes in the logic where made. To answer your question, the `__` comes from the method `aliasField()` which produces `TableName__fieldName`